collect_app/src/main/java/org/odk/collect/android/widgets/AnnotateWidget.java

```diff
@@ -65,6 +65,9 @@ public AnnotateWidget(Context context, QuestionDetails prompt) {
         imageCaptureHandler = new ImageCaptureHandler();
         setUpLayout();
         addCurrentImageToLayout();
+        if (binaryName == null || imageView == null || imageView.getVisibility() == GONE) {
```

Comment:
It would be good to factor it out to a separate method like adjustAnnotateButtonaAailability(0 for example.